transferui/src/main/res/layout/activity_list_transfer_destination.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/collapsing_header_size"
+        android:theme="@style/AppTheme.AppBarOverlay">
+
+        <com.google.android.material.appbar.CollapsingToolbarLayout
```

Comment:
I don't think activity needs collapsing toolbar or any toolbar since it is defined in the fragment already on how the toolbar looks like, can we please have a look on the design in the ticket?
![Screen Shot 2019-07-25 at 3 24 05 PM](https://user-images.githubusercontent.com/18428954/61912902-408edf80-aef0-11e9-890c-f695a9a3e866.png)

Reply:
removed it here https://github.com/hyperwallet/hyperwallet-android-ui-sdk/pull/84